atomic-app/templates/hpa-app.yaml

```diff
@@ -19,12 +20,41 @@ spec:
     - type: Resource
```

Comment:
What if we use:
```
{{- with .Values.app.autoscaling.metrics }}
      metrics:
        {{- toYaml . | nindent 4 }}
{{- end }}
{{- with .Values.app.autoscaling.behavior }}
      behavior:
        {{- toYaml . | nindent 4 }}
{{- end }}
```

And then push all the configuration to the values file.  This allows maximum flexibility which I think would be a good thing here.